pdf2archive

```diff
@@ -104,6 +106,16 @@ if [ "$(which gs)" == "" ]; then
     exit
 fi
 
+#=====# VALIDATION #=====#
+
+validate() {
```

Comment:
It's probably better to split the java check into an independent function.

Reply:
Did so with [this](https://github.com/matteosecli/pdf2archive/pull/14/commits/3244ac699158a7a5fb62856a1ba56af93bddaa3b).